torch_xla/csrc/ops/ops.cpp

```diff
@@ -1137,4 +1137,43 @@ torch::lazy::NodePtr Selu(const XlaValue& input) {
                    input.xla_shape(), std::move(lower_fn));
 }
 
+torch::lazy::NodePtr DynamicExpand(const XlaValue& input,
```

Comment:
We should move this to a separate node file since this op is not trivial at all.